CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Early Semantic Versioning](https://docs.scala-lang.org/overviews/core/binary-compatibility-for-library-authors.html#recommended-versioning-scheme) in addition to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+* [#84](https://github.com/typelevel/sbt-tpolecat/pull/84) - An `advancedOption` overload with support for multiple argument options was added.
+
+### Changed
+
+* [#85](https://github.com/typelevel/sbt-tpolecat/pull/85) - `-Xcheckinit` was removed from the default option set. After discussion on issues [#10](https://github.com/typelevel/sbt-tpolecat/issues/10) and [#83](https://github.com/typelevel/sbt-tpolecat/issues/83) it has become apparent that although `-Xcheckinit` offers a lot of value for some users, it creates some very tricky problems for users that are writing async code. The initialization checking code this option produces introduces `@volatile` variables that can disguise memory visibility issues.
```

Comment:
Technically, as long as you test _and_ publish your code with `-Xcheckinit` then you won't have hidden memory visibility bugs, just performance issues. It's only a problem if you use it for testing, but then publish without it (removing all the extra memory barriers).

Reply:
Yeah that is true, I wonder how it affects things when (for example) a user specifies -Xcheckinit and has inlining enabled. Seems like there are a lot of possible interactions here depending upon the permutations of user vs library-specified options.